### ISSUE TITLE
My fork of pytorch-mask-rcnn: PyTorch 0.4, batch size > 1, Windows compatibility, etc

### ISSUE THREAD

**Issue by Britefury** [multimodallearning/pytorch-mask-rcnn#47]
I have made a fork of pytorch-mask-rcnn here:

https://github.com/Britefury/pytorch-mask-rcnn/tree/refactor     (note the refactor branch; not master)

It fixes a number of issues:

- batch size can be > 1
- compatible with PyTorch 0.4
- extensions (NMS etc) compile and run on Windows
- multi-GPU training possible (although may not work well)
- significant refactor; Mask R-CNN inherits Faster R-CNN inherits RPN, so you can choose which bits of the model you want
- Detectron compatibility; I dropped compatibility with Matterport weights and focused on compatibility with Detectron instead. You can load Detectron weights and have it generate results that are not quite correct, yet...
- quite a few other things I forget....

Unfortunately its not very well documented and the original Coco based examples are unlikely to work. There are some new examples though, but they need work.

Basically, I'm wondering if I call it a new project going on, or.... Would be interested to hear @lasseha's thoughts :)
